python/lsst/pipe/analysis/plotUtils.py

```diff
@@ -485,27 +485,29 @@ def plotCcdOutline(axes, butler, dataId, ccdList, tractInfo=None, zpLabel=None,
             ccd = ccd[-2] + "," + ccd[-1]
             ccdLabelStr = "R" + str(raft) + "S" + str(ccd)
         dataIdCopy[ccdKey] = ccd
-        calexp = butler.get("calexp", dataIdCopy)
-        dataRef = butler.dataRef("raw", dataId=dataIdCopy)
         # Check metadata to see if stack used was HSC
         metadata = butler.get("calexp_md", dataIdCopy)
         hscRun = checkHscStack(metadata)
         if zpLabel and (zpLabel == "MEAS_MOSAIC" or "MEAS_MOSAIC_1" in zpLabel):
+            # Unfortunate, but not easily avoided
+            calexp = butler.get("calexp", dataIdCopy)
+            dataRef = butler.dataRef("raw", dataId=dataIdCopy)
             applyMosaicResultsExposure(dataRef, calexp=calexp)
             wcs = calexp.getWcs()
         elif zpLabel and (zpLabel == "MMphotoCalib" or zpLabel == "JOINTCAL" or
                           "MMphotoCalib" in zpLabel or "JOINTCAL_1" in zpLabel):
-            wcs = dataRef.get("jointcal_wcs")
+            wcs = butler.get("jointcal_wcs", dataIdCopy)
         else:
-            wcs = calexp.getWcs()
-        w = calexp.getWidth()
-        h = calexp.getHeight()
+            wcs = butler.get("calexp_wcs", dataIdCopy)
+        bbox = butler.get("calexp_bbox", dataIdCopy)
+        w = bbox.getWidth()
+        h = bbox.getHeight()
         if zpLabel is not None:
             if hscRun and (zpLabel == "MEAS_MOSAIC" or "MEAS_MOSAIC_1" in zpLabel):
-                nQuarter = calexp.getDetector().getOrientation().getNQuarter()
+                nQuarter = reader.readDetector().getOrientation().getNQuarter()
```

Comment:
Where is `reader` defined? (is there no `_detector` component access?)

Reply:
It was from a previous iteration when I thought I needed it.  Thanks for the catch.